GVFS/GVFS.Platform.Mac/ProjFSKext.cs

```diff
@@ -73,5 +69,73 @@ public bool TryPrepareFolderForCallbacks(string folderPath, out string error, ou
 
             return true;
         }
+
+        private ProcessResult Bash(string cmd)
+        {
+            string escapedArgs = cmd.Replace("\"", "\\\"");
+
+            ProcessStartInfo startInfo = new ProcessStartInfo
+            {
+                FileName = "/bin/bash",
+                Arguments = $"-c \"{escapedArgs}\"",
+                RedirectStandardOutput = true,
+                RedirectStandardError = true,
+                UseShellExecute = false,
+                CreateNoWindow = true,
+            };
+
+            using (Process process = new Process())
+            {
+                process.StartInfo = startInfo;
+                string errors = string.Empty;
+                process.ErrorDataReceived += (sender, args) =>
+                {
+                    if (args.Data != null)
+                    {
+                        errors += args.Data + "\r\n";
+                    }
+                };
+
+                process.Start();
+                process.BeginErrorReadLine();
+                string output = process.StandardOutput.ReadToEnd();
+                process.WaitForExit();
+                return new ProcessResult(output, errors, process.ExitCode);
+            }
+        }
+
+        private bool TryLoadKext(ITracer tracer, string enlistmentRoot, out string error)
+        {
+            Console.WriteLine("Kernel extension not loaded.  Attempting to load...");
+            error = DriverName + " is not loaded. Make sure the driver is loaded and try again.";
+            ProcessResult loadKext = this.Bash("sudo kextutil /Library/Extensions/PrjFSKext.kext");
+            if (loadKext.ExitCode == 0)
+            {
+                error = null;
+                return true;
+            }
+            else if (loadKext.ExitCode == 27)

```

Comment:
Add const

Reply:
Not sure if we have a precident but I like seeing both the 0 and 27 hardcoded to consts.